### ISSUE TITLE
Share dropdown missing "Location"

### ISSUE THREAD

**Issue by licaon-kter** [siacs/Conversations#2172]
...but you can have it as a share action if you set it as default in Settings.

**Comment by iNPUTmice** [comment]
You need to have an app installed that provides that Intent. Like the Share Location Plugin or the Open Share Location Plugin.

**Comment by licaon-kter** [comment]
I know, I had to watch the log because #1525 is not fixed (as in F-Droid does not have such an app and you don't search for other apps anyway).

But let me rephrase then, either have them in both menus (share and Settings) or remove it from both, right now one can erroneously set it up in Settings and what's the point in that?

**Comment by licaon-kter** [comment]
Thanks for the commit.
